app.js

```diff
@@ -4,8 +4,50 @@ var cookieParser = require('cookie-parser');
 var fs = require('fs');
 var logger = require('morgan');
 const webpush = require('web-push');
-
 var indexRouter = require('./routes/index');
+const AdminBro = require('admin-bro')
+const AdminBroSequelize = require('admin-bro-sequelizejs')
+const AdminBroExpress = require('admin-bro-expressjs')
+const bcrypt = require('bcrypt')
```

Comment:
This is a nitpick: recommend using [agon2](https://www.npmjs.com/package/argon2)